PactConsumerSwiftTests/MatcherSpec.swift

```diff
@@ -66,7 +66,7 @@ class MatcherSpec: QuickSpec {
       }
 
       it("allows min to be specified") {
-        subject = Matcher.eachLike(arrayItem, min: 4)
+        subject = Matcher.eachLike(arrayItem, minValue: 4)
 
```

Comment:
Can we leave this as min and put in an exclusion? Changing the matcher's api will break any users who are relying on this matcher in their tests.